README.md

```diff
@@ -51,43 +73,51 @@ The plugin will emit an error if it detects that it it missed a `loadstart` even
+ * `contentchanged` (EVENT) -- Fires when a new content video has been loaded in the player (specially, at the same time as the `loadstart` media event for the new source). This means the ad workflow has restarted from the beginning. Your integration will need to trigger `adsready` again, for example. Note that when changing sources, the playback state of the player is retained: if the previous source was playing, the new source will also be playing and the ad workflow will not wait for a new `play` event.
 
-* `adsready` (EVENT) — Trigger this event after to signal that your integration is ready to play ads.
-* `adplaying` (EVENT) - Trigger this event when an ads starts playing. If your integration triggers `playing` event when an ad begins, it will automatically be redispatched as `adplaying`.
-* `adscanceled` (EVENT) — Trigger this event after starting up the player or setting a new video to skip ads entirely. This event is optional; if you always plan on displaying ads, you don't need to worry about triggering it.
-* `adserror` (EVENT) - Trigger this event to indicate that an error in the ad integration has ocurred and any ad states should abort so that content can resume.
-* `nopreroll` (EVENT) - Trigger this event to indicate that there will be no preroll ads. This event is optional, but can improve user experience. It is ignored if it is seen after `readyforpreroll`.
-* `nopostroll` (EVENT) - Trigger this event to indicate that there will be no postroll ad. Otherwise, contrib-ads will trigger an adtimeout event after content ends if there is no postroll.
-* `ads-ad-started` (EVENT) - Trigger this when each individual ad begins.
-* `contentresumed` (EVENT) - If your integration does not result in a "playing" event when resuming content after an ad, send this event to signal that content can resume. This was added to support stitched ads and is not normally necessary.
-* `ads.startLinearAdMode()` (METHOD) — Call this method to signal that your integration is about to play a linear ad. This method triggers `adstart` to be emitted by the player.
-* `ads.endLinearAdMode()` (METHOD) — Call this method to signal that your integration is finished playing linear ads, ready for content video to resume. This method triggers `adend` to be emitted by the player.
-* `ads.skipLinearAdMode()` (METHOD) — Call this method to signal that your integration has received an ad response but is not going to play a linear ad.  This method triggers `adskip` to be emitted by the player.
-* `ads.stitchedAds()` (METHOD) — Get or set the `stitchedAds` setting.
-* `ads.videoElementRecycled()` (METHOD) - Returns true if ad playback is taking place in the content element.
+Deprecated events:
 
-In addition, video.js provides a number of events and APIs that might be useful to you.
-For example, the `ended` event signals that the content video has played to completion.
+* `contentplayback` (EVENT) -- The `playing` media event has the same meaning and is far more reliable.
```

Comment:
Although I like these descriptions, they seem to fit better in the migration guide. I would then link to the most recent migration guide in this section

Reply:
Oh yeah, since we're removing it in this version we can move it from "deprecated" to the migration guide.